views/about.erb

```diff
@@ -0,0 +1,70 @@
+<html>
+<header>
+  <h1>About Me:</h1>
+  <p>
+    These are all the things I'm sure of.
+  </p>
+</header>
+
+<body>
+  <nav>
+  <ul>
+    <li><a href="./blog.erb">Blog</a></li>
+    <li><a href="./projects.erb">Projects</a></li>
+    <li><a href="./index.erb">Home</a></li>
+  </ul>
+  </nav>
+
+  <section>
+    <table border="1" style="font-family: helvetica; border-collapse: collapse; text-align: center">
+    <thead>
+      <tr>
+        <th colspan="2">
+          What you need to know
+        </th>
+    </thead>
+    <tr>
+      <td>
+        Born:
+      </td>
+      <td>
+        USA
+      </td>
+    </tr>
+    <tr>
+      <td >
+        Age:
+      </td>
+      <td>
+        25
+      </td>
+    </tr>
+    <tr>
+      <td >
+        Fav Food:
+      </td>
+      <td>
+        Tacos
+      </td>
+    </tr>
+    <tr>
+    <td>
+      Pet:
+    </td>
+    <td>
+      Guinea Pig
+    </td>
+  </tr>
+  </table>
+  </section>
+
+  <section>
+    <p>
+      <a href="https://www.youtube.com/watch?v=QH2-TGUlwu4"><img src = "https://www.facebook.com/photo.php?fbid=10153049224875468&set=t.504319324&type=3&theater" alt = "Claire"></a>
```

Comment:
We are used to putting spaces between the `=` in ruby but in HTML we don't add those spaces in HTML so `<img src="`